docs/source/branch.rst

```diff
@@ -0,0 +1,218 @@
+=================
+The Branches File
+=================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+When investigating a research question with the Vivarium framework, it usually becomes necessary to vary aspects of a
+model configuration in order to evaluate the uncertainty of model outputs or to explore different scenarios based on
+model parameters. Without any extra tooling this would require manually manipulating the configuration file and
+re-running for each desired change which would quickly get out of hand. The branches file helps us do this in a
+convenient way. This section will detail the common ways simulations are varied and the different aspects of a branches
+file that help us do this.
+
+Uncertainty
+-----------
+
+Generating uncertainty for results is a core tenant of IHME and this is no different for simulation science. We are
+primarily concerned with uncertainty from two sources in our models -- uncertainty surrounding the data that goes in to
+a model and uncertainty due to stochasticity. The branches file can help us explore these uncertainties easily by
+allowing us to vary the input draws from the Global Burden of Disease (GBD) used as well as the random seeds used by the
+simulation.
+
+Data Uncertainty
+^^^^^^^^^^^^^^^^
+Our simulations primarily rely on GBD results which are produced in draws to estimate uncertainty. In order to reflect
+the GBD's uncertainty surrounding their results in our simulations, we generally run simulations with a variety of
+output draws.
+
+.. note::
```

Comment:
extra return here?

Reply:
looks like it shows up different on my screen. It is a linewrap to stay under 120. It should render ok

Reply:
hmm the return on line 34 looks like it shows up on the rendered docs though